organizational/conduct.rst

```diff
@@ -0,0 +1,101 @@
+The Mission of Hacker Society
+=============================
+
+A time and place for people to learn about technology from each other, in order
```

Comment:
start off talking about what organization we're talking about. Later on, you use HacSoc w/o defining the abbreviation. You can argue that we all know what HacSoc is, but I understand that this is supposed to be a more formal declaration.
"Hacker Society (HacSoc) at Case Western Reserve University is a time and place blah blah blah"